### ISSUE TITLE
Feature request: add support for extra mouse buttons

### ISSUE THREAD

**Issue by mediochrea** [nylas/N1#1997]
Also known as Xbutton1 and Xbutton2.

**Comment by bengotow** [comment]
Hey! What would these mouse buttons do?

**Comment by mediochrea** [comment]
Browser-like back/forward navigation. Or if full history isn't an option, just exposing them for the keymap file would be enough.

**Comment by francislavoie** [comment]
👍 to this, I found it jarring that my mouse's back/forwards buttons didn't work. I don't like having to move my mouse to the top left corner to go back to the inbox view. For now, that's a dealbreaker for me.

**Comment by mediochrea** [comment]
@francislavoie A shitty workaround that I came up with is rebinding a mouse button to the Escape key through Autohotkey, but it wasn't very consistent with how one would think a Back button would function, so I ended up disabling it. A native support would be very much welcome, but it's been a year since I opened the issue so I guess it's not happening.

**Comment by iainheng** [comment]
Yeah. Mouse button to navigate back is very handy and it is definitely a good feature to have built-in.
